JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -55,31 +61,49 @@ private enum class Sections(val title: String) {
 }
 
 @Composable
-fun InterestsScreen(openDrawer: () -> Unit) {
+fun InterestsScreen(scaffoldState: ScaffoldState = remember { ScaffoldState() }) {
+    Scaffold(
+        scaffoldState = scaffoldState,
+        drawerContent = {
+            AppDrawer(
+                currentScreen = Screen.Interests,
+                closeDrawer = { scaffoldState.drawerState = DrawerState.Closed }
+            )
+        },
+        topAppBar = {
+            TopAppBar(
+                title = { Text("Interests") },
+                navigationIcon = {
+                    VectorImageButton(R.drawable.ic_jetnews_logo) {
+                        scaffoldState.drawerState = DrawerState.Opened
+                    }
+                }
+            )
+        },
+        bodyContent = {
+            val (currentSection, updateSection) = state { Sections.Topics }
+            IntrestsScreenBody(currentSection, updateSection)
```

Comment:
Typo in `IntrestsScreenBody` -> `InterestsScreenBody`

Reply:
+1